ceph-ansible-prs/config/definitions/ceph-ansible-prs.yml

```diff
@@ -24,11 +40,78 @@
       - ansible2.2-bluestore_dmcrypt_journal
       - ansible2.2-bluestore_dmcrypt_journal_collocation
     jobs:
-        - 'ceph-ansible-prs-{release}-{scenario}'
+        - 'ceph-ansible-prs-{release_before_luminous}-{scenario_before_luminous}'
+        - 'ceph-ansible-prs-{release_after_luminous}-{scenario_after_luminous}'
+
+- job-template:
+    name: 'ceph-ansible-prs-{release_before_luminous}-{scenario_before_luminous}'
```

Comment:
I think it will be easier to handle the logic on what to test in the ceph-ansible tests. This is just a template, and handling logic like this is going to be hard